packages/preview/isc-hei-report/0.1.3/isc_template_report.typ

```diff
@@ -0,0 +1,272 @@
+// Template for ISC reports at the School of engineering
+// v0.1.3 - mui 2024
+//
+// Missing features : 
+// - page and locations (above, under) references for figures not available yet
+
+// Fancy pretty print with line numbers and stuff
+#import "@preview/codelst:2.0.1": sourcecode 
+
+// Nice color boxes
+#import "@preview/showybox:2.0.1": showybox
+
+// Define our own functions
+#let todo(body, fill_color: yellow.lighten(50%)) = {
```

Comment:
This argument uses snake_case while the rest of the package seems to be kebab-cased. Maybe you want to make that uniform?

Reply:
Noted and changed, thanks.